third_party/skia/include/core/SkTypes.h

```diff
@@ -653,4 +638,19 @@ enum class SkBackingFit {
     kExact
 };
 
+#if defined(STARBOARD)
+typedef enum SkPmcolor { SkPmcolorIsRgba, SkPmcolorIsBgra } SkPmColor;
+
+inline SkPmcolor GetSkPmcolor() {
+    if (SK_PMCOLOR_BYTE_ORDER(R,G,B,A)) {
+        return SkPmcolorIsRgba;
+    } else if (SK_PMCOLOR_BYTE_ORDER(B,G,R,A)) {
+        return SkPmcolorIsBgra;
+    } else {
+        SkASSERTF(false,"SK shift values do not correspond to a supported byte order.");
+        return (SkPmcolor)NULL;
+    }
+}
+#endif
+
```

Comment:
Why is this moved?

Reply:
It uses skASSERTF which is defined earlier in this file. Skia code does not use DCHECK/CHECKs, so our earlier modification needed an update to avoid pulling in //base/logging